govcd/metadata_v2_test.go

```diff
@@ -274,6 +287,128 @@ func (vcd *TestVCD) TestCatalogItemMetadata(check *C) {
 	}
 
 	testMetadataCRUDActions(catalogItem, check, nil)
+	vcd.testMetadataIgnore(catalogItem, "catalogItem", catalogItem.CatalogItem.Name, check)
+}
+
+func (vcd *TestVCD) testMetadataIgnore(resource metadataCompatible, objectType, objectName string, check *C) {
```

Comment:
I really like how this test was implemented, but is it needed to check every case of metadata for every type of resource? Not against it, just to spark more of a discussion, as it is quite a lot of API calls.

Reply:
Ideally not, as the String metadata should behave the same as Boolean or Date types, but one never knows 😇

Reply:
cc: @Didainius @dataclouder

Reply:
I'd say that we could agree on it - maybe a middle ground - a way to skip long running tests (like `skipLong` tag in Terraform)

Reply:
Okay, I can agree on this approach.

Reply:
I've added the `skipLong` tag, although if it is used it will skip all metadata tests (as far as I know one can't use tags on specific tests, but only to the whole file).

Before the changes on this PR it took **1552.618s** to run and with the changes, **1792.050s** (from my local env, considering latency and so).

In Terraform what I did is using an environment variable to skip some of the multiple metadata cases, maybe it would be more fine-grained.